### ISSUE TITLE
Duplicating compressed redstone / carbon

### ISSUE THREAD

**Issue by Mateiaru** [MJRLegends/Space-Astronomy-Feedback-#493]
### Steps to Reproduce Issue
1. Make an infusing factory with 8 speed & 8 energy upgrades (not sure about the upgrades)
2. Make sure the factory has at least 1 compressed carbon / redstone / obsidian or osmium
3. Put 1 compressed carbon / redstone etc. (depending on what's in the factory) in the TOP-LEFT slot of the inventory
4. Shift-left click the compressed carbon / redstone
5. Instead of going in the factory, it goes to the hotbar duplicated.
Notes:
 - The hotbar must not have any compressed redstone, or it will simply stack with it (it can have stacks of it tho)
 - Shift-clicking more than 32 (including 64) will result in 64 only (so 40 wont make 80 but 64)

Video (sorry for HD):
https://www.youtube.com/watch?v=lgllIHrWJFk

***UPDATE***:
Heres a list of what can be duplicated (tested with factories that were full of that item (bar & slot):
 - compressed carbon, redstone, diamond, obsidian
 - osmium, coal, refined obsidian dust, diamond dust
 - salt (in chemical injection factory)
 - flint (in purifying factory)

So basically any item that can be put in a factory as secondary fuel / whatever can be duplicated using this method (no redstone dust for some reason however).

**Version of Mod Pack using:** 1.5.2

**Comment by MJRLegends** [comment]
Reported to the dev https://github.com/aidancbrady/Mekanism/issues/3686

**Comment by Mateiaru** [comment]
Update: It also works with compressed diamond & obsidian.

**Comment by Mateiaru** [comment]
IMPORTANT UPDATE: I added (hopefully) all possible items that can be duplicated.

**Comment by MJRLegends** [comment]
This has been added to the mod pack known issues so im closing it and we await mekanism devs reply

**Comment by MJRLegends** [comment]
Fixed in v1.5.3
